src/utils/dom.ts

```diff
@@ -37,6 +37,9 @@ export const attrUp = (el?: HTMLElement, attrs: ObjectAny = {}) =>
   el && el.setAttribute && each(attrs, (value, key) => el.setAttribute(key, value));
 
 export const isVisible = (el?: HTMLElement) => {
+  if (!el || !(el instanceof HTMLElement)) {
+    return false;
+  }
```

Comment:
This part can be removed as I added already a fix. `getClientRects?.()` was enough as a check